grammars/ruby.cson

```diff
@@ -77,7 +86,15 @@
         'name': 'entity.other.inherited-class.module.third.ruby'
       '8':
         'name': 'punctuation.separator.inheritance.ruby'
-    'match': '^\\s*(module)\\s+(([A-Z]\\w*(::))?([A-Z]\\w*(::))?([A-Z]\\w*(::))*[A-Z]\\w*)'
+    'match': '''(?x)
+      ^\\s*(module)\\s+
+      (
+        ([A-Z]\\w*(::))?
+        ([A-Z]\\w*(::))?
+        ([A-Z]\\w*(::))*
```

Comment:
😂  Me too… I wanted to remove them. But on second thought, this PR is already big enough, so I decided to do what I planned to do here—beautify things, to keep things simple. Maybe another PR?